### DISCUSSION TITLE
Probably I'm just missing something...

### DISCUSSION THREAD

**Discussion by imforumman** [floccusaddon/floccus#969] [Q&A] [answered]
Rather large private NextCloud installation running fine, Bookmarks add-on running also.

Firefox on Windows. Installed floccus. Synchronization ran well, I see the Booksmarks in the NC-add-on.

KiWi on Android. Installed floccus. Synchronizatino ran without error. I see the one Bookmarks folder ("Mobile bookmarks") alongside the folders which Firefox uploaded.

Both Floccus installations have a root folder in local and remote settings and both are set up not to exclude from other clients.

Upon further synchronizations nothing else happens, neither the bookmarks from Firefox ever reach Kiwi and the Kiwi-bookmarks never reach Firefox. For all I care, the booksmarks from Kiwi could have been deleted in the process, but I want those bookmarks from Firefox on Kiwi and I want to synch them back-and-forth from then on. (Like I used to with Firefox synch until the mobile Firefox was robbed of all my extensions.)

So, where am I missing something?

**Comment by marcelklehr** [comment, marked as answer]
Hey! I'm happy to help. So, the folders from firefox show up in Kiwi but not the bookmarks? I'm not too familiar with Kiwi -- Which main folders did you have in Kiwi before syning with floccus? Syncing the root folder sometimes doesn't work with specific browsers. You could try selecting the Mobile Bookmarks folder in Kiwi as your target folder instead.

**Comment by imforumman** [comment]
Wow, thanks for your quick answer.
The result is that the bookmark folders from both "clients" are on the server, but each client does not see the booksmarks coming from the other client (I can't check Firefox right now, as I'm not at home). At least that is the way I think it is.

If I delete a bookmark on the server by hand, Kiwi just re-uploads it on sync (faulty behavior).
If I delete a bookmark on Kiwi, sync will delete it on the server (expected behavior).

**Reply by marcelklehr** [reply]
> The result is that the bookmark folders from both "clients" are on the server, but each client does not see the booksmarks coming from the other client

I see. And your firefox bookmarks are on the server in "Bookmarks bar" folders etc., while your kiwi bookmarks are in a "Mobile bookmarks" folder on the server.

It is likely that Kiwi doesn't allow adding additional root folders besides Mobile Bookmarks. In this case, simply sync your bookmarks to the "Mobile Bookmarks" folder as suggested above.

> If I delete a bookmark on the server by hand, Kiwi just re-uploads it on sync (faulty behavior).

Do you have sync mode set to normal or push?

**Reply by imforumman** [reply]
Normal (merge) sync on all clients.

I'm wondering if Kiwi doesn't allow manipulation of bookmarks at all by a plugin...

**Reply by marcelklehr** [reply]
I think it does. From what I've gathered, there are people who seem to be fine with floccus on kiwi. :/

**Reply by imforumman** [reply]
Ok, so when using "Mobile bookmarks" as the root folder on Kiwi it works!!! May be worthwhile adding to the documentation.
Thank you so much this plugin and your work on the NC plugin.

**Comment by imforumman** [comment]
One feature request: I'm not sure if you are familiar with the "send tab to device" feature of firefox sync, I miss it very much. Often I will be looking at a link on my mobile, and wish I could continue to follow up the next time I'm sitting at my real computer, so I would just send that tab to that device. All it does, is open a tab with the URL you are looking at the next time it syncs booksmarks, kind of like a on-demand-tab-sync. Do you think it may be possible to add this feature some time in the future?

**Reply by marcelklehr** [reply]
This is sort of possible, although it's a bit more complex to set up. Instead of syncing to a local folder like Mobile Bookmarks, you can set up a new account and choose to sync the tabs, and instead of syncing them to the root folder on your nextcloud you can sync them to a subfolder on your nextcloud. Since you may not want to sync them all the time, you can disable auto-sync for the account and only push or pull as necessary (those are hidden beneath the "more" chevron for each account in the account overview)

![image](https://user-images.githubusercontent.com/986878/134955414-eff5458f-8c47-4bee-aeaa-069b618c6e08.png)
